socorro/processor/cache_manager.py

```diff
@@ -0,0 +1,378 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""
+This defines the DiskCacheManager application. It's designed to run as
+a standalone application separate from the rest of the processor.
+
+It keeps track of files in a directory and evicts files least recently
+used in order to keep the total size under a max number.
+
+It uses inotify to cheaply watch the files.
+
+To run::
+
+    $ /app/bin/run_processor_cache_manager.sh
+
+"""
+
+from collections import OrderedDict
+import datetime
+import logging
+import os
+import pathlib
+import sys
+import traceback
+
+from boltons.dictutils import OneToOne
+from fillmore.libsentry import set_up_sentry
+from fillmore.scrubber import Scrubber, SCRUB_RULES_DEFAULT
+from inotify_simple import INotify, flags
+import markus
+
+from socorro import settings
```

Comment:
I thought about having this take command line arguments instead of using settings. I'm not sure which is the better approach, but none of the other services in Socorro take command line arguments (now) so I decided to have it work like them.